source/matrix_free/evaluation_template_factory.inst.in

```diff
@@ -32,51 +32,4 @@ for (deal_II_dimension : DIMENSIONS;
       deal_II_dimension,
       deal_II_scalar_vectorized::value_type,
       deal_II_scalar_vectorized>;
-
-    template bool dealii::internal::FEFaceEvaluationFactory<
-      deal_II_dimension,
-      deal_II_scalar_vectorized::value_type,
-      deal_II_scalar_vectorized>::
-      gather_evaluate<1>(
-        const unsigned int,
-        const deal_II_scalar_vectorized::value_type *,
-        const MatrixFreeFunctions::ShapeInfo<deal_II_scalar_vectorized> &,
-        const MatrixFreeFunctions::DoFInfo &,
-        deal_II_scalar_vectorized *,
-        deal_II_scalar_vectorized *,
-        deal_II_scalar_vectorized *,
-        const bool,
-        const bool,
-        const unsigned int,
-        const unsigned int,
-        const std::array<unsigned int, 1>,
-        const std::array<unsigned int, 1>,
-        const unsigned int,
-        const MatrixFreeFunctions::DoFInfo::DoFAccessIndex,
-        const std::array<unsigned int, 1>,
-        const Table<2, unsigned int> &);
-
-    template bool dealii::internal::FEFaceEvaluationFactory<
-      deal_II_dimension,
-      deal_II_scalar_vectorized::value_type,
-      deal_II_scalar_vectorized>::
-      integrate_scatter<1>(
-        const unsigned int,
-        deal_II_scalar_vectorized::value_type *,
-        const MatrixFreeFunctions::ShapeInfo<deal_II_scalar_vectorized> &,
-        const MatrixFreeFunctions::DoFInfo &,
-        deal_II_scalar_vectorized *,
-        deal_II_scalar_vectorized *,
-        deal_II_scalar_vectorized *,
-        deal_II_scalar_vectorized *,
-        const bool,
-        const bool,
-        const unsigned int,
-        const unsigned int,
-        const std::array<unsigned int, 1>,
-        const std::array<unsigned int, 1>,
-        const unsigned int,
-        const MatrixFreeFunctions::DoFInfo::DoFAccessIndex,
-        const std::array<unsigned int, 1>,
-        const Table<2, unsigned int> &);
```

Comment:
The key goal of this PR is get rid of these. Once we have introduced a `InternalData` the code should become significant simpler.